### PR TITLE
Feature/3508 start otanalytics via configuration file

### PR DESCRIPTION
**Pull request by randy-seng**
OP#3508
OP#3872

### CODE REVIEW

File: tests/OTAnalytics/plugin_ui/test_cli.py [hunk 1]
```diff
@@ -144,101 +147,105 @@ def temp_section(test_data_tmp_dir: Path, otsection_file: Path) -> YieldFixture[
...
+    save_dir: str = "",
     save_name: str = "",
     save_suffix: str = "",
-    event_list_exporter: EventListExporter = AVAILABLE_EVENTLIST_EXPORTERS[
-        OTC_OTEVENTS_FORMAT_NAME
-    ],
-    count_interval: int = 1,
+    event_formats: str = DEFAULT_EVENTLIST_FILE_TYPE,
+    count_intervals: int = 1,
+    # event_formats: list[str] | None = None,
```
Comment (briemla) [inline tests/OTAnalytics/plugin_ui/test_cli.py:204]: Do we need the commented code?

File: tests/OTAnalytics/plugin_ui/test_cli.py [hunk 2]
```diff
@@ -144,101 +147,105 @@ def temp_section(test_data_tmp_dir: Path, otsection_file: Path) -> YieldFixture[
...
+    # count_intervals: list[int] | None = None,
     num_processes: int = DEFAULT_NUM_PROCESSES,
     logfile: str = str(DEFAULT_LOG_FILE),
     logfile_overwrite: bool = False,
-) -> CliArguments:
+) -> RunConfiguration:
+    # if event_formats is None:
```
Comment (briemla) [inline tests/OTAnalytics/plugin_ui/test_cli.py:210]: Do we need the commented code?

**Review by briemla — COMMENTED** [review]
Looks good so far, just a few questions.

File: tests/OTAnalytics/plugin_ui/test_cli.py (hunk 1 above)
Comment (randy-seng) [reply tests/OTAnalytics/plugin_ui/test_cli.py]: I simply forgot to implement the feature to allow the user to pass multiple arguments to `--event-format` and `count-interval`. These were comments pertaining to this.

**Review by briemla — APPROVED** [review]

**Pull request merged**